Sources/XcodeProj/Scheme/XCScheme+BuildAction.swift

```diff
@@ -84,11 +116,13 @@ extension XCScheme {
                     postActions: [ExecutionAction] = [],
                     parallelizeBuild: Bool = false,
                     buildImplicitDependencies: Bool = false,
-                    runPostActionsOnFailure: Bool? = nil) {
+                    runPostActionsOnFailure: Bool? = nil,
+                    buildArchitectures: Architectures = .matchRunDestination) {
```

Comment:
This is an interesting one

- New schemes created in the Xcode UI do indeed default to `.matchRunDestination`
- The current released version of `XcodeProj` doesn't explicitly write this attribute when creating new schemes (i.e. would be `.useTargetSettings`)

Having `matchRunDestination` as the new default for the `init` here would introduce a change in behaviour for existing clients of the library and all their generated schemes would go from `.useTargetSettings` > `.matchRunDestination` 🙈

```suggestion
                    buildArchitectures: Architectures = .useTargetSettings) {
```

Reply:
The current released version didn't have control over this attribute, so I thought it wouldn't affect existing clients as if it were a new feature.
However, I missed the fact that there may be clients who are using the current version's settings well. Thank you for your review.